prov/efa/src/rxr/rxr_init.c

```diff
@@ -200,7 +199,7 @@ int rxr_raw_addr_to_smr_name(void *ptr, char *smr_name)
 		return -errno;
 	}
 
-	ret = snprintf(smr_name, SMR_NAME_MAX, "%s_%04x_%08x_%04x",
+	ret = snprintf(smr_name, EFA_SHM_NAME_MAX, "%s_%04x_%08x_%04x",
```

Comment:
Can you add a check to return EINVAL if `ret >= EFA_SHM_NAME_MAX`? That way we won't hide bugs if the format string changes for whatever reason in the future.

Reply:
Sure. Good point!

added in new revision.